web/api/formatters/rrdset2json.c

```diff
@@ -163,3 +172,143 @@ void rrdset2json(RRDSET *st, BUFFER *wb, size_t *dimensions_count, size_t *memor
 
     rrdset_unlock(st);
 }
+#endif /* ENABLE_JSONC */
+
+
+#ifdef ENABLE_JSONC
+#define JSON_ADD_STRING(name, str, obj)                                                                                \
```

Comment:
We have an analogous definition in the file `web/api/formatters/charts2json.c`, please, let us centralize this an `JSON_ADD_INT` in a header.

Reply:
moved to common file